src/main/java/frc/utils/brakestate/BrakeStateManager.java

```diff
@@ -0,0 +1,28 @@
+package frc.utils.brakestate;
+
+
+import frc.utils.MotorSubsystem;
+
+import java.util.ArrayList;
+
+public class BrakeStateManager {
+
+	private static final ArrayList<MotorSubsystem> motorSubsystems = new ArrayList<>();
+
+	public static void add(MotorSubsystem motorSubsystem) {
```

Comment:
make add, get 2 runnables, one for coast and one for brake. and put them in 2 Lists (one for coast and one for brake.) and the calling functions will call the brake runnable, or the coast runnable.